src/DependencyInjection/EzSystemsPlatformHttpCacheExtension.php

```diff
@@ -35,4 +35,9 @@ public function prepend(ContainerBuilder $container)
         $container->prependExtensionConfig('fos_http_cache', $config);
         $container->addResource(new FileResource($configFile));
     }
+
+    public function getAlias()
+    {
+        return 'ez_platform_cache';
```

Comment:
`ez_platform_http_cache`, method can afaik be avoided by renaming this class to EzPlatformHttpCacheExtension which `getContainerExtension()` will then need to be adapted for.

Reply:
I don't think that will be better. If I rename class, then I think the most clean way should be to also override Bundle::getContainerExtensionClass() ( and not write logic concerning class name in getContainerExtension() ).

So IMO it is either to create a getAlias() or a getContainerExtensionClass() method.

Reply:
Replaced getAlias() with getContainerExtensionClass()